docs/validators/upgrades/v5.0-upgrade.md

```diff
@@ -0,0 +1,94 @@
+# Upgrade Guide v5.0
```

Comment:
Heads up, you need to branch off `main` rather than an existing feature branch to prevent this.